packages/frontend/prisma/schema.prisma

```diff
@@ -44,6 +44,9 @@ enum GroupType {
   ManyNouns
 }
 
+// The above code MUST match the schema in `packages/merkle_tree/prisma/schema.prisma
```

Comment:
@amirgamil I added this reminder because it wasn't obvious to me that the prefixes have to match (I thought the whole file).